3723/1/repo.rb

```diff
@@ -0,0 +1,27 @@
+require 'json'
+require 'yaml'
+require 'faraday'
+
+class Repo
+  attr_reader :list
+
+  def initialize
+    @list = []
+  end
+
+  def write_gem(info)
```

Comment:
You use this method only in `take_repo`, so it might be private.

Reply:
Done